plugins/android-extensions/android-extensions-compiler/build.gradle.kts

```diff
@@ -33,7 +35,7 @@ dependencies {
 
     testRuntime(intellijPluginDep("junit"))
 
-    robolectricClasspath(commonDep("org.robolectric", "robolectric"))
+    robolectricClasspath("org.robolectric:robolectric:4.0")
```

Comment:
Could you please try to change `versions.robolectric` in the root `build.gradle.kts` instead?

Reply:
Certainly, thanks for the pointer.